src/device_facade.py

```diff
@@ -149,7 +149,13 @@ def click(self):
             else:
                 import uiautomator2
                 try:
-                    self.viewV2.click(UI_TIMEOUT_LONG)
+                    if mode=="center":
+                        self.viewV2.click(UI_TIMEOUT_LONG, offset=(random.uniform(0.1,0.9),random.uniform(0.1,0.9)))
+                    elif mode=="left":
+                        self.viewV2.click(UI_TIMEOUT_LONG, offset=(random.uniform(0.25,0.6),random.uniform(0.1,0.9)))
+                        #0.25 to avoid pressing on stories if present
+                    elif mode=="right":
+                        self.viewV2.click(UI_TIMEOUT_LONG, offset=(random.uniform(0.6,0.9),random.uniform(0.1,0.9)))
```

Comment:
Isn't that enough to randomize it only around the center (with a short range)?
You had to add a condition 0.25 to avoid pressing in a story of a list of users. But click() is rather generic method. It might not be a good idea to keep adding exceptions/conditions here.

Reply:
About the "center" mode it was intended as "whole" button mode.
Does it make sense?
if you generic generic click() call, then you can press on the whole button..

About the "left" mode do you have suggestion? I mean, I can look if there is the stories available and choose between "left" and "center" (which is not the whole button, but a short-range center click)
Or I can only press in the middle